blog/2023-07-29-airtable/airtable.md

```diff
@@ -10,8 +10,8 @@ tags: [airtable,guide]
 When you using a pre-build template with Airtable, you will need to configure `airtable_table_name`, `airtable_base_id` and `airtable_token_name`. And you also need to grant flows.network to access your Airtable via the API key provided by Airtable.
 
 * `airtable_table_name`: the name of the table you want to add data, which you can find on Airtable.
```

Comment:
I would really like to see the overview of these settings / variables to be presented as a table instead of a bullet list. The table could have two columns: Name and Value, which correspond to the labels on the UI.

Reply:
Great idea! Sorry for missing this comment. Will create another PR to add it.